align_anything/configs/template.py

```diff
@@ -147,6 +426,114 @@ def format_sample(self, raw_sample: dict[str, Any]) -> dict[str, Any]:
             'image': Image.open(requests.get(image_file, stream=True).raw),
         }
 
+@register_template('LLAVA-CC3M')
+class LLAVA_CC3M:
+    user_prompt: str = 'USER: \n{input}'
+    assistant_prompt: str = '\nASSISTANT: {output}'
+    
+    def format_sample(self, raw_sample: dict[str, Any], path: str=None) -> dict[str, Any]:
+        raw_conversations = raw_sample['conversations']
+        question = raw_conversations[0]['value']
+        answer = raw_conversations[1]['value']
+        image = raw_sample['image']
+
+        text = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output=answer)}"
+        )
+
+        prompt = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output='')}"
+        )
+        image_file = os.path.join(path, 'images', image)
+
+        return {
+            'text': text,
+            'prompt': prompt,
+            'image': Image.open(image_file),
+        }
+
+@register_template('AudioCaps')
+class AudioCaps:
+    user_prompt: str = 'USER: {input}'
+    assistant_prompt: str = '\nASSISTANT: {output}'
+    def format_sample(self, raw_sample: dict[str, Any], path: str=None) -> dict[str, Any]:
+        caption = raw_sample['caption']
+        audiocap_path = raw_sample['audiocap_path']
+        question = random.choice(AUDIO_QUESTIONS)
+        
+        text = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output=caption)}"
+        )
+
+        prompt = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output='')}"
+        )
+        audio, sample_rate = torchaudio.load(os.path.join(path, f"{audiocap_path}"))
+        if audio.shape[0] == 2:
+            audio = audio.mean(dim=0, keepdim=True)
+        return {
+            'text': text,
+            'prompt': prompt,
+            'audio': audio.squeeze().tolist(),
+            'sample_rate': sample_rate
+        }
+
+@register_template('LibriSpeech')
+class LibriSpeech:
+    user_prompt: str = 'USER: {input}'
+    assistant_prompt: str = '\nASSISTANT: {output}'
+    def format_sample(self, raw_sample: dict[str, Any], path: str=None) -> dict[str, Any]:
+        caption = raw_sample['text'].lower()
+        question = random.choice(SPEECH_QUESTIONS)
+        
+        text = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output=caption)}"
+        )
+
+        prompt = (
+            f'{self.user_prompt.format(input=question)}'
+            f"{self.assistant_prompt.format(output='')}"
+        )
+        audio = raw_sample['audio']['array']
+        return {
+            'text': text,
+            'prompt': prompt,
+            'audio': audio,
+            'sample_rate': 48000  
```

Comment:
the sample_rate seems fixed, is it right?

Reply:
has fixed